eng/scripts/Language-Settings.ps1

```diff
@@ -167,7 +167,7 @@ function Get-dotnet-AdditionalValidationPackagesFromPackageSet {
   # The targetedFiles needs to filter out anything in the ExcludePaths
   # otherwise it'll end up processing things below that it shouldn't be.
   foreach ($excludePath in $diffObj.ExcludePaths) {
-    $targetedFiles = $targetedFiles | Where-Object { -not $_.StartsWith($excludePath) }
+    $targetedFiles = $targetedFiles | Where-Object { -not $_.StartsWith($excludePath.TrimEnd("/") + "/") }
```

Comment:
It is feeling like some of this matching code should be in common. @JimSuplizio I don't know your plans on making this configuration driven but when you do this might be a good set of code to share.

Reply:
Right now, I'm just trying to correct the error where it'll overmatch. Configuration driven isn't really going to affect this particular piece of the code. That's just going to move things like startsWithPrefixes/additional validation modules to something that's not hard coded in the ps1 file, meaning it'll load a json or some such.

Reply:
The configuration itself is language specific but the code that will load the configuration and try to match files against it should be able to be common.